api/src/main/scala/io/phdata/rest/WorkspaceController.scala

```diff
@@ -198,7 +199,12 @@ class WorkspaceController[F[_]: Sync: Timer: ContextShift: ConcurrentEffect](
               case e: Throwable =>
                 logger.error(s"Failed to add member to workspace $id: ${e.getLocalizedMessage}", e).pure[F]
             }
-            _ <- emailService.newMemberEmail(id, memberRequest).value
+            emailResult = try {
+              emailService.newMemberEmail(id, memberRequest).value
+            } catch{
+              case e: Exception =>
+                logger.error((s"Failed to send welcoming message, due to this reason: $e"))
+            }
```

Comment:
Are you sure that if sending a mail fails the whole adding member fails as well?

Reply:
We are pretty sure. It's intermittent. My theory is that if the email fails to send before the endpoint returns it fails. The email sent in a fiber, so that should be possible.